dao/src/main/java/cz.muni.fi.pa165.soccermanager/dao/PlayerDaoImpl.java

```diff
@@ -0,0 +1,45 @@
+package cz.muni.fi.pa165.soccermanager.dao;
```

Comment:
this file should not be here, check if your branch starts in master and not in my player-dao

if it starts in my branch (and  I think so :) )
rebase your branch on master and force push